### PR TITLE
Coalition Hand to Hand weapons

### PR DESCRIPTION
**Pull request by Brick63**
add weapons to make the capturing of coalition ships more diffcult

enforcer confrontation gear@2x
enforcer riot staff@2x

[heliarch wep assets.zip](https://github.com/endless-sky/endless-sky/files/3731162/heliarch.wep.assets.zip)

### CODE REVIEW

File: data/coalition outfits.txt [hunk 1]
```diff
@@ -221,6 +221,28 @@ outfit "Large Steering Module"
...
+outfit "Enforcer Riot Staff"
+	category "Hand to Hand"
+	licenses
+		Heliarch
+	cost 130000
+	thumbnail "outfit/enforcer riot staff"
+	"capture attack" 2.3
+	"capture defense" 3.5
+	"unplunderable" 1
+	description "Made to deal with the occasional dissidents that pop up around coalition space in a non lethal manor, these see minimal use in shipbound combat but are there in the event a dissident gets aboard"
```
Comment (toilethinges) [inline data/coalition outfits.txt]: non lethal manor --> non-lethal manner

Comment (Brick63) [reply data/coalition outfits.txt]: thanks ill fix that

File: data/coalition outfits.txt [hunk 2]
```diff
@@ -221,6 +221,28 @@ outfit "Large Steering Module"
...
+outfit "Enforcer Riot Staff"
+	category "Hand to Hand"
+	licenses
+		Heliarch
+	cost 130000
+	thumbnail "outfit/enforcer riot staff"
+	"capture attack" 2.3
+	"capture defense" 3.5
+	"unplunderable" 1
+	description "Made to deal with the occasional dissidents that pop up around coalition space in a non lethal manner, these see minimal use in shipbound combat, but are there in the event a dissident gets aboard."
```
Comment (Amazinite) [inline data/coalition outfits.txt]: Capitalize Coalition.

File: data/coalition outfits.txt [hunk 3]
```diff
@@ -221,6 +221,28 @@ outfit "Large Steering Module"
 	"turning heat" 8.2
 	description "By focusing on small, modular engines rather than the massive capital ship drives that most other species develop, the Coalition loses some of the potential efficiency for very large ships, but gains a considerable amount of flexibility."
 
+outfit "Enforcer Confrontation Gear"
+	category "Hand to Hand"
+	licenses
+		Heliarch
+	cost 230000
+	thumbnail "outfit/enforcer confrontation gear"
```
Comment (Amazinite) [inline data/coalition outfits.txt]: With this description, I'd be more interested in having the image be the one that Becca made. The staff looks great, though.

File: data/coalition outfits.txt [hunk 4]
```diff
@@ -221,6 +221,28 @@ outfit "Large Steering Module"
...
+	"unplunderable" 1
+	description "The peak of Heliarch combat engineering, this weapon uses condensed blasts of energy to dispatch any who dare threaten the Coalition safety."
+
+outfit "Enforcer Riot Staff"
+	category "Hand to Hand"
+	licenses
+		Heliarch
+	cost 130000
+	thumbnail "outfit/enforcer riot staff"
+	"capture attack" 2.3
```
Comment (Amazinite) [inline data/coalition outfits.txt]: Drop this to 2 so it's lesser than the Pug staff, then drop the cost to 71k.

File: data/coalition outfits.txt [hunk 5]
```diff
@@ -221,6 +221,28 @@ outfit "Large Steering Module"
 	"turning heat" 8.2
 	description "By focusing on small, modular engines rather than the massive capital ship drives that most other species develop, the Coalition loses some of the potential efficiency for very large ships, but gains a considerable amount of flexibility."
 
+outfit "Enforcer Confrontation Gear"
+	category "Hand to Hand"
+	licenses
+		Heliarch
+	cost 230000
```
Comment (Amazinite) [inline data/coalition outfits.txt]: Drop the cost to round about 172k.

**Comment by Amazinite** [comment]
Since this will be giving the Heliarch H2H weapons, you should also drop the default attack and defense of the Heliarch government. I'd say set their default attack from 3 to 1.5 and their default defense from 4 to 3.

File: data/coalition outfits.txt [hunk 6]
```diff
@@ -221,6 +221,28 @@ outfit "Large Steering Module"
...
+outfit "Enforcer Riot Staff"
+	category "Hand to Hand"
+	licenses
+		Heliarch
+	cost 97000
+	thumbnail "outfit/enforcer riot staff"
+	"capture attack" 2.0
+	"capture defense" 3.5
+	"unplunderable" 1
+	description "Made to deal with the occasional dissidents that pop up around Coalition space in a non lethal manner, these see minimal use in shipbound combat, but are there in the event a dissident gets aboard."
```
Comment (Pointedstick) [inline data/coalition outfits.txt]: `these see` -> these weapons see`

`but are there` -> `but are carried anyway`

**Pull request merged**